gammapy/maps/wcsnd.py

```diff
@@ -599,3 +600,50 @@ def smooth(self, radius, kernel='gauss', **kwargs):
         image = copy.copy(self)
         image.data = data
         return image
+
+    def make_cutout(self, position, radius, margin=None, mode="strict"):
+        """
+        Create a cutout of a WcsNDMap around a given direction.
+
+        Parameters
+        ----------
+
+        position : `~astropy.coordinates.SkyCoord`
+            Center position of the cutout box
+        radius : tuple of `~astropy.coordinates.Angle`
+            Angular sizes of the box
+        margin : `~astropy.coordinates.Angle`, optional
+            Additional safety margin. If specified must be same length as radius
+        mode : {'trim', 'partial', 'strict'}
+            Mode option for Cutout2D,  see http://docs.astropy.org/en/stable/api/astropy.nddata.utils.Cutout2D.html 
```

Comment:
Use this instead of a URL to link to the Astropy docs:
```
`~astropy.nddata.utils.Cutout2D`
```
This is using intersphinx to make the links.